meta-balena-common/recipes-containers/balena-supervisor/balena-supervisor/balena-supervisor.service

```diff
@@ -4,7 +4,8 @@ Requires=\
     resin\x2ddata.mount \
     balena-device-uuid.service \
     os-config-devicekey.service \
-    bind-etc-balena-supervisor.service
+    bind-etc-balena-supervisor.service \
+    extract-balena-ca
```

Comment:
```
Feb 09 09:47:03 systemd[1]: /lib/systemd/system/balena-supervisor.service:3: Failed to add dependency on extract-balena-ca, ignoring: Invalid argument
Feb 09 09:47:03 systemd[1]: /lib/systemd/system/balena-supervisor.service:5: Failed to add dependency on extract-balena-ca, ignoring: Invalid argument
```

Reply:
thanks @klutchell it seems to be missing a `.service`. Strange that the dependency was working, need to revisit this.